src/internal/_filterObj.js

```diff
@@ -0,0 +1,11 @@
+var _baseCopy = require('./_baseCopy');
+
+module.exports = function _filterObj(fn, obj) {
+  var result = {};
+  for (var key in obj) {
```

Comment:
this will descend the whole proto chain, yes? maybe use `R.keys` here instead, as in `mapObj`